packages/node_modules/@webex/internal-plugin-team/src/team.js

```diff
@@ -123,39 +126,43 @@ const Team = WebexPlugin.extend({
   /**
    * Retrieve a single team
    * @param {Object|Team~TeamObject} team
-   * @param {string} team.id
+   * @param {string} team.url
    * @param {Object} options
    * @param {boolean} options.includeTeamConversations
    * @param {boolean} options.includeTeamMembers
    * @returns {Promise} Resolves with the requested team
    */
-  get(team, options) {
-    options = options || {};
-
+  get({url}, options = {}) {
+    if (!url) {
+      throw Error('`team.url` is required');
+    }
     const params = Object.assign({
       includeTeamConversations: false,
       includeTeamMembers: false
     }, options);
 
-    return this._inferTeamUrl(team)
-      .then(() => this.webex.request({
-        uri: team.url,
-        qs: params
-      }))
+    return this.webex.request({
+      uri: url,
+      qs: params
+    })
       .then((res) => this._recordUUIDs(res.body)
         .then(() => res.body));
   },
 
   /**
    * Get the list of conversations for a particular team
    * @param {TeamObject} team
+   * @param {String} team.url
    * @returns {Promise} Resolves with an array of conversations
    */
-  listConversations(team) {
-    return this._inferTeamUrl(team)
-      .then(() => this.webex.request({
-        uri: `${team.url}/conversations`
-      }))
+  listConversations({url}) {
```

Comment:
Interesting; I haven't seen this in a method header before.

Reply:
`url` is the only property I needed from the object.